scripts/lib/cluster

```diff
@@ -80,6 +80,34 @@ function log_operator_state() {
     done
   done
   kubectl describe pods --namespace="$namespace" >> "$log_file" 2>&1
+
+  # Print out logs for all pods in the operator namespace
+  local operator_pods="$(kubectl get pods --namespace="$namespace" -o=name)"
+  for pod in ${operator_pods}; do
+    kubectl logs "$pod" --namespace="$namespace" >> "$log_file" 2>&1 || true
+  done
+}
+
+function log_olm_state() {
+  local log_file="$1"
+
+  local olm_namespace="$2"
+
+  echo -e "\nolm pods describe\n" >> "$log_file"
+  kubectl describe pods --namespace="$olm_namespace" >> "$log_file" 2>&1 || true
+  echo -e "\n---\n" >> "$log_file"
+  echo -e "\nolm-operator logs\n" >> "$log_file"
+  kubectl logs $(get_first_matching_pod olm-operator-) --namespace="$olm_namespace" >> "$log_file" 2>&1 || true
+  echo -e "\n---\n" >> "$log_file"
+  echo -e "\ncatalog-operator logs\n" >> "$log_file"
+  kubectl logs $(get_first_matching_pod catlog-operator-) --namespace="$olm_namespace" >> "$log_file" 2>&1 || true
+  echo -e "\n---\n" >> "$log_file"
+}
+
+function get_first_matching_pod() {
+  local prefix="$1"
+
+  echo $(kubectl get pods -o name | awk '/$prefix/ {print $1;exit}')
```

Comment:
```suggestion
  echo $(kubectl get pods -o name | awk "/$prefix/ {print $1;exit}")
```
Bash variables are not evaluated in string literals (`'...'`).

Reply:
Using double quotes causes `$1` to be bash completed as well even though it should be the result of the regex match. I've change the line to use awk pattern matching with the prefix set as an awk variable